### ISSUE TITLE
Ionic Resources splash too large.

### ISSUE THREAD

**Issue by Tyler-Darby** [ionic-team/ionic-cli#2780]
When generating a splash with `ionic cordova resources -s` I get the following error:

```

**Comment by giltotherescue** [comment]
Convert the file to a PNG so the file size is smaller. Remember to remove the original file from the directory so it knows to use the smaller PNG file.

**Comment by Tyler-Darby** [comment]
@giltotherescue I've done this with a PNG and a JPEG. Took it as small as 1.7mb and still no luck.

**Comment by giltotherescue** [comment]
@Tyler-Darby Did you remember to remove the original PSD or AI file from the directory, so that it doesn't try to upload that version?

**Comment by Tyler-Darby** [comment]
@giltotherescue Yes, completely out of the project. I'm going to load up a blank project and test it again.

**Comment by Tyler-Darby** [comment]
Okay, it seems to sort of work in a new project. It hangs up partially through generation every time, though. ex: 

```
✔ Collecting resource configuration and source images - done!
✔ Filtering out image resources that do not need regeneration - done!
✔ Uploading source images to prepare for transformations - done!
⠏ Generating platform resources: 12/25 complete
```
